doc/nrf/releases/release-notes-latest.rst

```diff
@@ -12,6 +12,29 @@ Changelog
 
 See the `master branch section in the changelog`_ for a list of the most important changes.
 
+
+sdk-mcuboot
+===========
+
+The MCUboot fork in |NCS| contains all commits from the upstream MCUboot repository up to and including ``???``, plus some |NCS| specific additions.
+
+The following list summarizes the most important changes inherited from upstream MCUboot:
+
+  * Fixed an issue where a left-over image trailer was sometimes not erased after reset.
+  * Added a ``CONFIG_BOOT_INTR_VEC_RELOC`` option to relocate interrupts to application.
+  * Fixed single image compilation with serial recovery.
+  * Added support for single-image applications (see :option:`CONFIG_SINGLE_IMAGE_DFU`).
+  * Added a ``CONFIG_BOOT_SIGNATURE_TYPE_NONE`` option to disable the cryptographic check of the image.
+  * Reduced the minimum number of members in packet for serial recovery.
```

Comment:
What packet are we talking about? Is that clear enough?

Reply:
smp packet